app/qml/PositionMarker.qml

```diff
@@ -11,7 +12,62 @@ Item {
     property QgsQuick.PositionKit positionKit
     property color baseColor: InputStyle.highlightColor
     property bool withAccuracy: true
+    property int interval: 200 // Interval of direction marker updates in ms
+    property real threshold: 3 // threshold used to minimized direction marker updates (otherwise shaking)
+    property real speedLimit: 15  // Over speed limit, directions depends on direction of movement
+    property real direction: -1
+    property real groundSpeed: -1
```

Comment:
does this need to be a property? if not petter to just have it as a local "var" in onTriggered

Reply:
yes, can be local. used for debug in main.qml.